sentry_sdk/integrations/graphene.py

```diff
@@ -17,7 +20,7 @@
 
 
 if TYPE_CHECKING:
-    from typing import Any, Dict, Union
+    from typing import Any, Dict, Generator, Union
```

Comment:
```suggestion
    from collections.abc import Generator
    from typing import Any, Dict, Union
```

`typing.Generator` is a [deprecated alias](https://docs.python.org/3/library/typing.html#typing.Generator) to `collections.abc.Generator`